modules/carousel/Carousel.tsx

```diff
@@ -353,18 +375,156 @@ const Carousel: React.FC<ICarouselProps & WithStylesProps<Theme, ICarouselStyles
+        }
+
+        while (currentScroll >= indexScroll) {
+          indexScroll += childrenSize;
+          indexCount += 1;
+        }
+
+        const highOffset = indexScroll - currentScroll;
+        const lowOffset = currentScroll - (indexScroll - childrenSize);
+        let chosenOffset = indexScroll;
+
+        if (highOffset > lowOffset) {
+          chosenOffset -= childrenSize;
+          indexCount -= 1;
+        }
+
+        if (chosenOffset >= scrollBaseReference && Math.abs(chosenOffset - currentScroll) > 2) {
+          // eslint-disable-next-line no-param-reassign
+          scrollableElementRef.current.scrollLeft = chosenOffset;
+          setCenteredItemInd(indexCount);
+        }
+      }
+    };
+    // eslint-disable-next-line mdx/no-unused-expressions
+    scrollableElementRef?.current?.addEventListener("scroll", debounce(eventListener, 300));
+  }, [
+    alignmentType,
+    autoFocus,
+    calculateChildrenSize,
+    items.length,
+    scrollBaseReference,
+    scrollableElementRef
+  ]);
+
+  /**
+   * This useEffect darkens the elements beside the centered one.
+   * As the user scrolls, this calculates how dark should be based on
+   * the distance from the center to that item.
+   * The furthest from the center, the darker will get
+   */
+  useEffect(() => {
+    if (alignmentType !== "centered" || !scrollableElementRef?.current || !darkenFurthestItems) {
+      return;
+    }
+
+    const element = scrollableElementRef.current;
+
+    const darkenEffectFunc = () => {
+      element.childNodes.forEach((item: any, index: number) => {
+        if (index === 0 || index === element.childNodes.length - 1) {
+          return;
+        }
+
+        let dummyDiv = item.querySelector(".carousel-dummy-overlay");
+        if (!dummyDiv) {
+          // eslint-disable-next-line no-param-reassign
+          item.style.position = "relative";
+
+          const div = document.createElement("div");
+          div.className = "carousel-dummy-overlay";
+          div.style.position = "absolute";
+          div.style.top = "0";
+          div.style.left = "0";
+          div.style.right = "0";
+          div.style.bottom = "0";
+          item.appendChild(div);
+
+          dummyDiv = item.querySelector(".carousel-dummy-overlay");
+        }
+
+        const offsetDiff =
+          Math.abs(
+            element.scrollLeft +
+              element.clientWidth / 2 -
+              (item.offsetLeft + calculateChildrenSize() / 2 - marginBetweenItems / 2)
+          ) / element.clientWidth;
+        // eslint-disable-next-line no-param-reassign
+        dummyDiv.style.display = "block";
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion, no-param-reassign
+        dummyDiv.style.backgroundColor = `rgba(7, 7, 7, ${offsetDiff < 0.1 ? 0 : offsetDiff})`;
+        // eslint-disable-next-line no-param-reassign
+        item.style.borderStyle = "hidden";
+
+        if (offsetDiff < 0.1) {
```

Comment:
Given this condition where you set display: none, maybe the condition in line 486 (backgroundColor) doesn't matter?